### ISSUE TITLE
Maintainers

### ISSUE THREAD

**Issue by YoshiYoshi360** [manton/JSONFeed#173]
I would like to make some PRs for this project but I noticed that it has been unmaintained for about a year. Is their anyway someone can join to become an additional maintainer for the project?

**Comment by manton** [comment]
Sorry, I’ve been thinking about how best to handle this. Part of the problem with all the pull requests is we migrated the website to a different host and there’s not a great workflow for going from merging them in to actually deploying the change.

I think I’ll go through the PRs and manually update the site where I can. Then we can reevaluate if more needs to be done.

Thanks.

**Comment by matklad** [comment]
As a data point, I am evaluating if I should switch my blog to json feed. 

I absolutely love the format itself. It is simple, and it is done. It reveals the fundamental simplicity of the thing, hitherto concealed by over engineered XML.

But there are two major sticking points for me:

The landing page at https://www.jsonfeed.org/ is underwhelming. There’s no clear link to the spec on it, there’s no example! Instead, it is a list of two blog posts from years ago.

And, yeah, the state of repository is a bit of a disarray, with tones of open PRs and issues, some recentish, with no response. To clarity, I don’t think there needs to be much _work_ done here, but responding to some of the issues with “thanks for the suggestion, but no, the thing is finishes” would help me understand that it is indeed finished rather that abandoned.

Great work on the spec itself, I wish I’ve learned about it earlier!

**Comment by matklad** [comment]
> Part of the problem with all the pull requests is we migrated the website to a different host

Consider hosting on GitHub pages (keeping domain name, of course). It feels like it’ll long term minimize “ops” pain.

**Comment by DesignSkeptic** [comment]
Agreed. The HUGO website can be deployed statically to GitHub in a new repo.

**Comment by FelicianoTech** [comment]
I've been thinking about this for awhile and since there's still no progress here, I'm forking the website. I will pull in a bunch of these assorted repos into one place, get the website updates/add more information, etc. In order to not spend any money at the moment, I'll host it as a subdirectory of one of my Linodians project until I figure out where to go from there.

If you're interested in the new website, you can view the new webpages here: https://www.linodians.com/jsonfeed/ That is a direct link and I'm not linking it from the main navigation yet. The website is built with Hugo.

If you're interested in contributing, the GitHub project is: `https://github.com/linodians/www.linodians.com` and the source directory for the JSON Feed section is `https://github.com/linodians/www.linodians.com/tree/trunk/src/content/jsonfeed`. I also made a list of tasks that I want to tackle so far for the restarted project here: https://github.com/linodians/www.linodians.com/issues/10

---

To the original authors, my contact email is on my profile, get in touch if you're upset, or if you'd like to contribute as well. I love what you've made and I just just want to see it flourish, not wither away.

**Comment by manton** [comment]
There has actually been some movement on this recently. Just last week we deployed some changes to the JSON Feed Validator website.

I don’t think a fork is necessary. Better would be for people to be able to contribute and have everything in one place so users aren’t confused.

**Comment by manton** [comment]
I’ll follow up in more detail tomorrow, but just wanted to post a quick comment. I’m glad people are interested in helping get the word out about JSON Feed. Thanks!

**Comment by manton** [comment]
Thanks. Moving the site to Micro.blog wasn't really about control, just was easier for me to manage with everything in one place with my other sites. (Micro.blog is fine for static sites, not just blogs.)

I agree that managing it in GitHub is probably more accessible for everyone.

**Comment by matklad** [comment]
I've been subscribed to all activity on this repo for a while. While there was a flurry of work around the time of fork announcement, it seems that not much was happening since (modulo a bunch of comments from a particular GitHub user).

From my observer's perspective, fork of the _website_ is 100% justified at this point. 

Though, @FelicianoTech, I strongly encourage you to make it a proper self-contained project, not affiliated with linodians.com. For long-term viability, it's imperative that code, ownership, and branding of JSON feed website isn't entangled with any other project. Otherwise I feel we are at risk of repeating exactly the same story where no one is empowered to make improvements, because the thing is tied into some unrelated infrastructure :-) 

I would also strongly encourage to not fork the _spec itself_ and just freezing it instead. Given the quality of the spec, I'd strongly prefer that all changes, if any are needed (!!) are approved by original authors (if you do fork, imo it'd be a good idea to also invite original maintainers). 

I would also recommend using GitHub+GitHub pages or https://codeberg.org for storing the source of truth for the website source code, and org permissions. 

But also, it's your fork and work, so don't take the above too seriously :-)

**Comment by manton** [comment]
There are things going on that are not yet visible in the GitHub activity. For example, I talked to the IndieWeb founders a few weeks ago about transferring everything to the IndieWeb's GitHub organization. I think that alone will resolve some of the concerns here. Forking the website isn't really going to help anything and will just fragment the documentation.

**Comment by ad732** [comment]
@manton any updates on the IndieWeb situation?
